css/index.css

```diff
@@ -101,17 +138,45 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes div {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
-
```

Comment:
Good job using parent/child overrides to change only what is needed to be changed for the child.  In this case, just the color for each individual box.